lib/bindings/src/breez_sdk_liquid.udl

```diff
@@ -313,35 +325,53 @@ dictionary GetInfoResponse {
 };
 
 dictionary PrepareSendRequest {
-    string invoice;
+    string destination;
+    u64? amount_sat;
+};
+
+[Enum]
+interface SendDestination {
+    LiquidAddress(LiquidAddressData address_data);
+    Bolt11(LNInvoice invoice);
 };
 
 dictionary PrepareSendResponse {
-    string invoice;
+    SendDestination destination;
     u64 fees_sat;
 };
 
+dictionary SendPaymentRequest {
+    PrepareSendResponse prepare_response;
+};
+
 dictionary SendPaymentResponse {
     Payment payment;
 };
 
-dictionary PrepareReceivePaymentRequest {
-    u64 payer_amount_sat;
+enum PaymentMethod {
+    "Lightning",
+    "BitcoinAddress",
+    "LiquidAddress",
 };
 
-dictionary PrepareReceivePaymentResponse {
-    u64 payer_amount_sat;
+dictionary PrepareReceiveRequest {
+    u64? amount_sat;
```

Comment:
Why was this renamed from `payer_amount_sat`? If that's still what it represents, it should remain unchanged IMO.